scripts/splitClasses.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs')
+
+const classes = JSON.parse(
+  fs.readFileSync('../src/assets/classes.json').toString()
+)
+
+/**
+ * Splits the category labels between a given number of instances.
+ * 
+ * If there more instances than categories, categories are cyclically reused.
+ */
+const main = () => {
+  const nInstances = process.argv[3] || 1
+  const current = process.argv[2] || nInstances
+  const nClasses = process.argv[4] || 1
+
+  let keys = Object.keys(classes)
+
+  const keysUsed = nInstances * nClasses
+
+  let result = []
+  for(let i=0; i<nClasses; i++) {
+    result.push(keys[((current - 1) * nClasses + i) % keys.length])
```

Comment:
I'm not comfortable with this script: its logic is heavily dependent on populate.sh, but it looks like it can be called in standalone. 
I guess you made this script because it was a bit tricky to do it in bash inside populate.sh?
I think I would prefer that you refactor this script a bit: 
- call it from populate before the for loop, with the following args: nInstances, nClasses
- add a for...i loop on nInstance: `current` is now the loop index
- save the results as an array that you can access from the populate.sh for loop